app/styles/login.scss

```diff
@@ -0,0 +1,3 @@
+.login {
+  text-align: -webkit-center;
```

Comment:
Is the `-webkit` prefix really needed?

Reply:
it's getting all elements to be centered, other options all not centering everything

Reply:
It know what it does, but we don't need the `-webkit` [vendor prefix](https://developer.mozilla.org/en-US/docs/Glossary/Vendor_Prefix). This is a standard css property which is implemented the same across the latest major browsers.